<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>KoDocumentInfoAuthorWidget</class>
 <widget class="QWidget" name="KoDocumentInfoAuthorWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>493</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="4" column="0" >
      <widget class="QLabel" name="lblEmailDesc" >
       <property name="text" >
        <string>Email:</string>
       </property>
       <property name="buddy" >
        <cstring>leEmail</cstring>
       </property>
      </widget>
     </item>
     <item row="9" column="1" >
      <widget class="QLineEdit" name="lePostal" />
     </item>
     <item row="8" column="0" >
      <widget class="QLabel" name="lblStreetDesc" >
       <property name="text" >
        <string>Street:</string>
       </property>
       <property name="buddy" >
        <cstring>leStreet</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="0" >
      <widget class="QLabel" name="lblPhoneWork" >
       <property name="text" >
        <string>Telephone (work):</string>
       </property>
       <property name="buddy" >
        <cstring>lePhoneWork</cstring>
       </property>
      </widget>
     </item>
     <item row="10" column="0" >
      <widget class="QLabel" name="lblCityDesc" >
       <property name="text" >
        <string>City:</string>
       </property>
       <property name="buddy" >
        <cstring>leCity</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="lePosition" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="lblTitleDesc" >
       <property name="text" >
        <string>Title:</string>
       </property>
       <property name="buddy" >
        <cstring>leTitle</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="lblInitialsDesc" >
       <property name="text" >
        <string>Initials:</string>
       </property>
       <property name="buddy" >
        <cstring>leInitials</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="0" >
      <widget class="QLabel" name="lblFaxDesc" >
       <property name="text" >
        <string>Fax:</string>
       </property>
       <property name="buddy" >
        <cstring>leFax</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="leTitle" />
     </item>
     <item row="10" column="1" >
      <widget class="QLineEdit" name="leCity" />
     </item>
     <item row="5" column="1" >
      <widget class="QLineEdit" name="lePhoneHome" />
     </item>
     <item row="9" column="0" >
      <widget class="QLabel" name="lblPostalDesc" >
       <property name="text" >
        <string>Postal code:</string>
       </property>
       <property name="buddy" >
        <cstring>lePostal</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="lblPositionDesc" >
       <property name="text" >
        <string>Position:</string>
       </property>
       <property name="buddy" >
        <cstring>lePosition</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="1" >
      <widget class="QLineEdit" name="leStreet" />
     </item>
     <item row="7" column="1" >
      <widget class="QLineEdit" name="leFax" />
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="lblCompanyDesc" >
       <property name="text" >
        <string>Company:</string>
       </property>
       <property name="buddy" >
        <cstring>leCompany</cstring>
       </property>
      </widget>
     </item>
     <item row="11" column="1" >
      <widget class="QLineEdit" name="leCountry" />
     </item>
     <item row="6" column="1" >
      <widget class="QLineEdit" name="lePhoneWork" />
     </item>
     <item row="5" column="0" >
      <widget class="QLabel" name="lblPhoneHome" >
       <property name="text" >
        <string>Telephone (home):</string>
       </property>
       <property name="buddy" >
        <cstring>lePhoneHome</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QLineEdit" name="leEmail" />
     </item>
     <item row="3" column="1" >
      <widget class="QLineEdit" name="leCompany" />
     </item>
     <item row="11" column="0" >
      <widget class="QLabel" name="lblCountryDesc" >
       <property name="text" >
        <string>Country:</string>
       </property>
       <property name="buddy" >
        <cstring>leCountry</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="leInitials" />
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="lblAuthor" >
       <property name="text" >
        <string/>
       </property>
       <property name="buddy" >
        <cstring>leFullName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leFullName" />
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="pbLoadKABC" >
       <property name="text" >
        <string>&amp;Load From Address Book</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbDelete" >
       <property name="text" >
        <string>&amp;Delete Personal Data</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
